scripts/state-sync.sh

```diff
@@ -14,54 +14,99 @@ then
     exit 1
 fi
 
-# Use argument as home directory if provided, else default to ~/.celestia-app
+usage() {
+  echo "Usage: $0 [--network <mainnet|arabica|mocha|local>] [single_node_home (for local only)]"
+  exit 1
+}
+
+NETWORK="mainnet"
+SINGLE_NODE_HOME="${HOME}/.celestia-app"
+
+# Parse args
 if [ $# -ge 1 ]; then
-  SINGLE_NODE_HOME="$1"
-else
-  SINGLE_NODE_HOME="${HOME}/.celestia-app"
+  if [ "$1" = "--network" ]; then
+    if [ $# -ge 2 ]; then
+      NETWORK="$2"
+      if [ "$NETWORK" = "local" ] && [ $# -ge 3 ]; then
+        SINGLE_NODE_HOME="$3"
+      fi
+    else
+      usage
+    fi
+  elif [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+    usage
+  fi
 fi
 
-CHAIN_ID="test"
-KEY_NAME="validator"
-KEYRING_BACKEND="test"
-COINS="1000000000000000utia"
-DELEGATION_AMOUNT="5000000000utia"
-CELESTIA_APP_HOME="${HOME}/.celestia-app-state-sync"
-CELESTIA_APP_VERSION=$(celestia-appd version 2>&1)
-GENESIS_FILE="${CELESTIA_APP_HOME}/config/genesis.json"
-FEES="500utia"
-RPC="0.0.0.0:26657"
+case "$NETWORK" in
+  mainnet)
+    CHAIN_ID="celestia"
+    SEEDS="e6116822e1a5e283d8a85d3ec38f4d232274eaf3@consensus-full-seed-1.celestia-bootstrap.net:26656,cf7ac8b19ff56a9d47c75551bd4864883d1e24b5@consensus-full-seed-2.celestia-bootstrap.net:26656"
+    RPC="https://celestia-rpc.polkachu.com:443"
+    CELESTIA_APP_HOME="${HOME}/.celestia-app"
+    ;;
+  arabica)
+    CHAIN_ID="arabica-11"
+    SEEDS="827583022cc6ce65cf762115642258f937c954cd@validator-1.celestia-arabica-11.com:26656,74e42b39f512f844492ff09e30af23d54579b7bc@validator-2.celestia-arabica-11.com:26656,00d577159b2eb1f524ef9c37cb389c020a2c38d2@validator-3.celestia-arabica-11.com:26656,b2871b6dc2e18916d07264af0e87c456c2bba04f@validator-4.celestia-arabica-11.com:26656"
+    RPC="https://rpc.celestia-arabica-11.com:443"
+    CELESTIA_APP_HOME="${HOME}/.celestia-app"
+    ;;
+  mocha)
+    CHAIN_ID="mocha-4"
+    SEEDS="5d0bf034d6e6a8b5ee31a2f42f753f1107b3a00e@celestia-testnet-seed.itrocket.net:11656"
+    RPC="https://celestia-testnet-rpc.itrocket.net:443"
+    CELESTIA_APP_HOME="${HOME}/.celestia-app"
```

Comment:
[nit] order this case statement in the same order as the networks (Arabica, then Mocha, then Mainnet)

Reply:
[blocking] CELESTIA_APP_HOME is identical in all of these cases so it can be extracted.